tf-modules/archive/api.tf

```diff
@@ -148,13 +150,25 @@ data "aws_iam_policy_document" "private_api_policy_document" {
 }
 
 resource "aws_api_gateway_rest_api" "api" {
```

Comment:
I'm wondering if a more explicit name is called for here now  (e.g. ngap_api, api_in_ngap, ?), rather than an implicit meaning and then specifying "other".      Not worth holding this up for, but something to consider.

Reply:
I agree that it would be good to rename, but unfortunately, we can't rename this resource - because if we do then it will be deleted/re-created by Terraform, which will then wipe out the custom changes made to our resource policy for the API gateway, which is precisely what this resource is trying to avoid.